doc/_read-me.txt

```diff
@@ -0,0 +1,14 @@
+Organization of these folders
+
+Th file  doc_xx.hs it is the configuration for java help by language.
+Th file map_xx.jhm there are the configuration of the help tree
```

Comment:
Typo: "Th" -> "The"
Better wording: "In addition, there is a map_??.jhm file to define the help tree."